vertical-pod-autoscaler/pkg/features/features.go

```diff
@@ -43,6 +43,12 @@ const (
 	// alpha: v1.4.0
 	// components: admission-controller, updater
 
+	// alpha: v1.5.0
+	// components: admission-controller, updater
+
+	// CPUStartupBoost enables the CPU startup boost feature.
+	CPUStartupBoost featuregate.Feature = "CPUStartupBoost"
```

Comment:
```suggestion
	// alpha: v1.5.0
	// components: admission-controller, updater

	// CPUStartupBoost enables the CPU startup boost feature.
	CPUStartupBoost featuregate.Feature = "CPUStartupBoost"
	
	// alpha: v1.4.0
	// components: admission-controller, updater
	
```

This flag got placed between the comments of InPlaceOrRecreate and the InPlaceOrRecreate flag itself.